lib/tape/ansible_runner.rb

```diff
@@ -87,11 +97,16 @@ def exec_ansible(playbook, args)
     Kernel.exec(cmd)
   end
 
-  def enforce_roles_path!
+  
+  def enforce_roles_path!     
     Dir.mkdir('.tape') unless Dir.exists?('.tape')
-    File.open('.tape/ansible.cfg', 'w') do |f|
+
+    File.open("#{local_dir}/.tape/ansible.cfg", 'w') do |f|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.